buildenv/jenkins/common/pipeline-functions.groovy

```diff
@@ -677,7 +677,7 @@ def setup_pull_request() {
        Jenkins test extended <platform>
        It was tested*/
 
-    def PARSED_BY_NEWLINE_COMMENT = params.ghprbCommentBody.split(/\\r?\\n/)
+    def PARSED_BY_NEWLINE_COMMENT = ghprbCommentBody.split(/\\r?\\n/)
     for (COMMENT in PARSED_BY_NEWLINE_COMMENT) {
```

Comment:
`PARSED_BY_NEWLINE_COMMENT` and `COMMENT`  are local variables. Please use lower case notation and rename COMMENT (e.g. line) as there is also a local variable `comment`.

Reply:
Updated the 2 functions I touched. Launching compile and test builds to prove those changes.